packages/@react-spectrum/datepicker/test/DatePicker.test.js

```diff
@@ -33,7 +33,7 @@ function getTextValue(el) {
 }
 
 function expectPlaceholder(el, placeholder) {
-  expect(getTextValue(el)).toBe(placeholder);
+  expect(getTextValue(el).replace(' ', ' ')).toBe(placeholder);
```

Comment:
What exactly is this replace for?

Reply:
see https://github.com/adobe/react-spectrum/pull/4945/files#r1322233532